src/coreComponents/constitutive/relativePermeability/RelativePermeabilityBase.cpp

```diff
@@ -105,6 +111,22 @@ void RelativePermeabilityBase::allocateConstitutiveData( dataRepository::Group &
   resizeFields( parent.size(), numConstitutivePointsPerParentIndex );
 }
 
+void RelativePermeabilityBase::updateTrappedPhaseVolFraction( arrayView2d< real64 const, compflow::USD_PHASE > const & phaseVolFraction ) const
+{
+
+  arrayView2d< real64, compflow::USD_PHASE > phaseTrapped = m_phaseTrapped.toView();
+
+  localIndex const numElems = phaseVolFraction.size( 0 );
+  integer const numPhases = numFluidPhases();
+  forAll< parallelDevicePolicy<> >( numElems, [=] GEOSX_HOST_DEVICE ( localIndex const ei ) {
```

Comment:
```suggestion
  forAll< parallelDevicePolicy<> >( numElems, [=] GEOSX_HOST_DEVICE ( localIndex const ei ) 
  {
```
to follow the GEOSX style with curly brackets.